shopify/resources/fulfillment.py

```diff
@@ -13,6 +14,24 @@ def complete(self):
     def open(self):
         self._load_attributes_from_response(self.post("open"))
 
+    def update_tracking(self, tracking_info, notify_customer):
+        fulfill = FulfillmentV2()
+        fulfill.id = self.id
+        self._load_attributes_from_response(fulfill.update_tracking(tracking_info, notify_customer))
+
 
 class FulfillmentOrders(ShopifyResource):
     _prefix_source = "/orders/$order_id/"
+
+class FulfillmentV2(ShopifyResource):
```

Comment:
I'm not sure I follow the idea of the `V2` nomenclature, is there a particular reason we can't just do this in `Fulfillment`? If the class is absolutely necessary, we might want to consider renaming it to something that better conveys what it does.

Reply:
I don't _believe_ we can do it in the original `Fulfillment` class because that class points to `/orders/$order_id/fulfillment/$fulfillment_id`, and the endpoint in this case is only `fulfillment/$fulfillment_id`. I would also prefer to do it in 1 class, but I'm not quite sure how..
The `V2` nomenclature is from [shopify_api](https://github.com/Shopify/shopify_api/blob/master/lib/shopify_api/resources/fulfillment_v2.rb) (that is used by [Fulfillment.rb](https://github.com/Shopify/shopify_api/blob/master/lib/shopify_api/resources/fulfillment.rb)) in a similar manner. Thought I would keep it consistent between the two libs

Reply:
All of that makes perfect sense, thanks for the context! I'm still not a fan of calling it `V2` but I agree we should keep them consistent.

Not saying we should do this, but I wonder if it wouldn't be cleaner to change `_prefix_source` for that particular method rather than adding a whole new class for it.

Reply:
I tried manipulating `_prefix_source` but wasn't successful with my attempt. I do believe the convention of changing prefixes is to override the classmethod `_prefix` depending on passed in options, but in this case the usage is after we define a Fulfillment object already :/
Thus new class

Reply:
Nothing wrong with this approach, I was just thinking aloud there!